Source/websocket/WebSocketLink.cpp

```diff
@@ -138,7 +132,7 @@ namespace Web {
                 result += usedSize;
             }
 
-            if (((_controlStatus & (REQUEST_CLOSE | REQUEST_PING | REQUEST_PONG)) != 0) && ((result + 1) < maxSendSize)) {
+            if (((_controlStatus & (REQUEST_CLOSE | REQUEST_PING | REQUEST_PONG)) != 0) && (result + ((_setFlags & MASKING_FRAME) ? 6 : 2) < maxSendSize)) {
```

Comment:
small remark but to make it more inline with our normal coding could you please make it ((_setFlags & MASKING_FRAME) != 0) we do not advocate the use of implicit conversion to boolean